### ISSUE TITLE
C2ISTAR access is required to use all support modules.

### ISSUE THREAD

**Issue by Taxen0** [ALiVEOS/ALiVE.OS#198]
When assigning which item is required for the Player Support modules (C2ISTAR, Combat, Logistics) the item specified in the C2ISTAR module will always be needed in addition to the other ones to get access.

example:
(module = item required to use)
C2ISTAR = compass
Combat Support = watch
Logistics = GPS

- If only the compass (C2ISTAR) is equipped access to the correct commands in the commander menu is granted. (correct behavior)
- If only the watch (Combat Support) and / or GPS (Logistics) is equipped the commander menu don't show up at all. (incorrect behavior)
- If the compass (C2ISTAR) is equipped together with the watch (Combat Support) and / or GPS (Logistics) the correct options show up in the commander menu. (correct behavior)

Also, having the Alive_tablet (Combat Support Tablet) always counts as having access to C2ISTAR options and behaves the same way as a compass in the above example would. Not sure if this is intended or not.

If you remove the C2ISTAR module from the map the watch (Combat Support) and GPS (Logistics) correctly add / remove the options from the commander menu as you expect.

A repo mission using the above settings and only Alive / cba is attached, simply hit the alive key and add / remove items from the inventory to reproduce.
https://www.dropbox.com/s/ski1rfvt9uy6v5k/C2ISTAR_repo.Stratis.zip?dl=0 
(for some reason I was not allowed to upload here even though it was a .zip)

**Comment by friznit** [comment]
Actually know about this for some time - frankly it's a bit of a bugger's muddle but in order to fix a bug from a few patches ago we had to make C2ISTAR the "master", so it currently overrides Combat Spt and Log.  The undesirable side effect is that you cannot have access to Combat Support only without also having access to C2ISTAR.  And to make things worse, CS doesn't appear to like playing solo anymore without C2ISTAR module placed.

Combat Support has been undergoing a rewrite for a long time (ahem...) so it's due to be fixed one day.

The ALiVE_tablet does indeed give you immediate and full access to C2ISTAR.  It's what the item was originally intended for.

Good bug report though, so will keep this one open for reference - thank!

**Comment by Gunny1979** [comment]
What was the bug that needed C2ISTAR to be the master?

I believe I have a solution to this issue, but would like to see what the original bug was so that I don't  inadvertently  resurrect it.

**Comment by friznit** [comment]
Don't have a record of it.  Something HH fixed, but it was directly related to this.  I would suggest first step is undoing the hardcoding of C2ISTAR to the alive_tablet.  I know it was intended that way originally but it's misleading.

**Comment by friznit** [comment]
From HH: the original problem was that you didnt have access to CS and support modules anymore if C2ISTAR was NOT placed, which led to unneeded script load and other issues IIRC

Ad.: - If only the watch (Combat Support) and / or GPS (Logistics) is equipped the commander menu don't show up at all. (incorrect behavior)....There should be menu items for the Support modules. And if C2ISTAR is placed the Support menu entries should be moved accordingly.
--------------
Basically, ideal behaviour should be that you can each module (Cbt Spt, Log, C2ISTAR) should be independently enabled by the item specified in that module.

**Comment by Gunny1979** [comment]
Fix in

**Comment by Taxen0** [comment]
Great work! Love the dedication you guys have, best modding team I've hade experience with =)
